CHANGELOG.md

```diff
@@ -3,6 +3,50 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+### Added
+- Include try-convert version in upgrade-assistant output [(#358)](https://github.com/dotnet/upgrade-assistant/pull/358)
+- Add check so ensure file can be loaded. [(#346)](https://github.com/dotnet/upgrade-assistant/pull/346)
+- Add a command to select a different current project [(#343)](https://github.com/dotnet/upgrade-assistant/pull/343)
+- Add a step to update package dependecies before TFM change [(#342)](https://github.com/dotnet/upgrade-assistant/pull/342)
+- Add IComponentIdentifier and ProjectComponents.AspNetCore [(#340)](https://github.com/dotnet/upgrade-assistant/pull/340)
+- Replace BinaryFormatter.UnsafeDeserialization [(#339)](https://github.com/dotnet/upgrade-assistant/pull/339)
+- Be sure to include log exception information if upgrade steps fail [(#335)](https://github.com/dotnet/upgrade-assistant/pull/335)
+- Add flag to exclude MSBuild runtime libraries [(#327)](https://github.com/dotnet/upgrade-assistant/pull/327)
+- Add package map entries for configuration and EF [(#324)](https://github.com/dotnet/upgrade-assistant/pull/324)
+- Include try-convert as part of build [(#322)](https://github.com/dotnet/upgrade-assistant/pull/322)
+- Update ready checks to apply per-project instead of solution-wide [(#309)](https://github.com/dotnet/upgrade-assistant/pull/309)
+- More extensibility samples and docs [(#305)](https://github.com/dotnet/upgrade-assistant/pull/305)
+- Add support to load VB project files [(#299)](https://github.com/dotnet/upgrade-assistant/pull/299)
+- Add trouble-shooting tips for installing with invalid/auth NuGet sources [(#291)](https://github.com/dotnet/upgrade-assistant/pull/291)
+- Add a sample demonstrating how to make a custom upgrade step [(#283)](https://github.com/dotnet/upgrade-assistant/pull/283)
+- Add option to provide entry-point and make non-interactive mode visible [(#282)](https://github.com/dotnet/upgrade-assistant/pull/282)
+- Add a way to verify upgrade readiness [(#279)](https://github.com/dotnet/upgrade-assistant/pull/279)
+- Allow controlling file/console logging independently [(#278)](https://github.com/dotnet/upgrade-assistant/pull/278)
+- Add SetPropertyValue to IProjectFile and MSBuildProject.File [(#274)](https://github.com/dotnet/upgrade-assistant/pull/274)
```

Comment:
This is an internal change that won't change the user experience, so we can probably leave it off.